### DISCUSSION TITLE
How to Do an Encrypted Dual Boot Using Windows 11 and Linux Mint - Mathis Gauthey's Website

### DISCUSSION THREAD

**Discussion by giscus** [mathisgauthey/mathisgauthey.github.io#2] [_]
# How to Do an Encrypted Dual Boot Using Windows 11 and Linux Mint - Mathis Gauthey's Website

How to Do an Encrypted Dual Boot Using Windows 11 and Linux Mint Objective Our goal is to create a completely independant encrypted dual boot setup for Linux Mint and Windows 11. How is it independant ? We&rsquo;ll make sure that GRUB (Linux Bootloader) will install itself on the Linux Drive, and won&rsquo;t be on your Windows drive with Windows Bootloader. That means you&rsquo;ll be able to remove any of the two drives and use them on any other machine you want.

https://mathisgauthey.github.io/how-to-do-an-encrypted-dual-boot-using-windows-11-and-linux-mint/

**Comment by gregoire-marie** [comment]
Very nicely presented, was most useful.
Thanks bud

**Comment by sfwtopoulos** [comment]
Thanks a lot!
